aws/import_aws_codebuild_project_test.go

```diff
@@ -0,0 +1,31 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAWSCodeBuildProject_importBasic(t *testing.T) {
+	resourceName := "aws_codebuild_project.test"
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSCodeBuildProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCodeBuildProjectConfig_basic(rName),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
```

Comment:
I'm guessing this was copy-paste from another resource's testing? 😄  The attributes `force_destroy` and `acl` do not exist for this resource, so the whole `ImportStateVerifyIgnore` bit can be removed.

Reply:
Oh, I'm sorry. This is my first code-reading terraform, so I referenced s3 bucket or something...
There is no acl for code build indeed, I'm embarrassed. 😓